### PR TITLE
Touching bloody objects will make your hands/gloves bloody.

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR makes your hands or gloves covered in blood if you touch bloodied objects. You will not spread blood, it will just be on you. This will also make all items you take bloodied without spending blood spread points.

## Why It's Good For The Game

More bloody evidence is good. Touching something bloody or touching something with bloodied hands should leave an evidence and not just fingerprints.

## Testing

Compiled, touched bloody skrells.

## Changelog
:cl:
tweak: Touching a bloody object will make your hands/gloves bloody. Hitting someone with your fist while they are bleeding will cover you in blood. Touching any item with bloodied hands/gloves will make it bloodied.
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -268,7 +268,7 @@
 	var/safe_from_poison = FALSE
 	strip_delay = 20
 	put_on_delay = 40
-	var/transfer_blood = FALSE
+	var/transfer_blood = 0
```
Comment (lewcc) [inline code/modules/clothing/clothing.dm:271]: Auto-doc this, please

File: code/modules/detective_work/detective_work.dm [hunk 2]
```diff
@@ -3,12 +3,29 @@
 /atom/proc/add_fibers(mob/living/carbon/human/M)
 	if(M.gloves && isclothing(M.gloves))
 		var/obj/item/clothing/gloves/G = M.gloves
-		if(G.transfer_blood > 1) //bloodied gloves transfer blood to touched objects
-			if(add_blood(G.blood_DNA, G.blood_color)) //only reduces the bloodiness of our gloves if the item wasn't already bloody
-				G.transfer_blood--
-	else if(M.bloody_hands > 1)
-		if(add_blood(M.blood_DNA, M.hand_blood_color))
+		if(isitem(src) && G.blood_DNA)
```
Comment (lewcc) [inline code/modules/detective_work/detective_work.dm:6]: Is there any way you could rework this to avoid doing what amounts to an `istype(src)` check?

**Review by lewcc — CHANGES_REQUESTED** [review]

File: code/modules/detective_work/detective_work.dm (hunk 2 above)
Comment (HMBGERDO) [reply code/modules/detective_work/detective_work.dm]: replaced with variable easy_to_spill_blood on atom. No istype check now, just extra variable on all atoms in game, i hope it works better

**Review by lewcc — APPROVED** [review]

**Comment by leboucliervert** [comment]
@HMBGERDO would it be possible for you to make it so soap cannot get bloody ? My OCD really does not like it.

**Comment by HMBGERDO** [comment]
> @HMBGERDO would it be possible for you to make it so soap cannot get bloody ? My OCD really does not like it...

changed it here
https://github.com/ParadiseSS13/Paradise/pull/26048

**Comment by grandalff** [comment]
Now the thing is, everyones hands are bloodied with god knows whose blood, because you touch a single bloody item it spreads like a plague to every other item you touch. Even alt clicking boxes spreads it. It, in all honestly, hampers the gathering of evidence.

**Pull request merged**